doc/dev/developer_guide/basic-workflow.rst

```diff
@@ -295,6 +295,56 @@ to change the `Issue tracker`_ status to "Resolved". Some issues may be
 flagged for backporting, in which case the status should be changed to
 "Pending Backport" (see the `Backporting`_ chapter for details).
 
+See also `Merging`_ for more information on merging.
+
+Proper Merge Commit Format
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This is the most basic form of a merge commit::
+
+       doc/component: title of the commit 
+
+       Reviewed-by: Reviewer Name <rname@example.com>
+
+This consists of two parts:
+
+#. The title of the commits of the pull request to be merged.
+#. The name and email address of the reviewer. Enclose the reviewer's email 
+   address in angle brackets.
+
+Using .githubmap to Find a Reviewer's Email Address
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+If you cannot find the email address of the reviewer on his or her github
+page, you can look it up in the **.githubmap** file, which can be found in
+the repository at **/ceph/.githubmap**.
+
+Using "git log" to find a Reviewer's Email Address
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+If you cannot find a reviewer's email address by using the above methods,
+you can search the git log for their email address. Reviewers are likely
+to have committed something before, and as long as they have committed
+something, the git log will probably contain their email address.
+
+Use the following command::
+
+.. prompt::
+
+     [branch under review]$ git log
```

Comment:
This doesn't render

Reply:
please see http://sbrunner.github.io/sphinx-prompt/ for more examples .